site/en/docs/privacy-sandbox/user-agent/snippets/index.md

```diff
@@ -0,0 +1,192 @@
+---
+layout: 'layouts/doc-post.njk'
+title: 'User-Agent Snippets'
+subhead: >
+  Code snippets to transform the current Chrome user-agent to the reduced format.
+description: >
+  Test your own sites and services against the reduced Chrome user-agent format by using a regular expression to transform the current Chrome user-agent to the reduced format.
+date: 2021-11-26
+authors:
+  - rowan_m
+---
+
+A collection of code snippets to let you transform the current Chrome user-agent
+string to the new reduced format. Please [submit versions for your preferred
+platforms and
+languages]https://github.com/GoogleChrome/developer.chrome.com//blob/main/site/en/docs/privacy-sandbox/user-agent/snippets/index.md)
+or let us know what would be useful to add!
+
+## Background
+Chrome is [reducing the information exposed in its user-agent
+string](https://developer.chrome.com/docs/privacy-sandbox/user-agent/). This
+will be happening incrementally over a period of time with the final state
+removing the OS version, device, and full browser version portions of the
+string.
+
+Here's an example of how that would look on a mobile device:
+
+{% Compare 'worse', 'old' %} <span style="font-family: monospace">Mozilla/5.0
+(Linux; Android <span style="background: #ef9a9a">12; Pixel 5</span>)
+AppleWebKit/537.36 (KHTML, like Gecko) Chrome/95.<span  style="background:
+#ef9a9a">0.4638.16</span> Mobile Safari/537.36</span> {% endCompare %}
+
+{% Compare 'better', 'new' %} <span style="font-family: monospace">Mozilla/5.0
+(Linux; Android <span style="background: #a5d6a7">10; K</span>)
+AppleWebKit/537.36 (KHTML, like Gecko) Chrome/95.<span style="background:
+#a5d6a7">0.0.0</span> Mobile Safari/537.36</span> {% endCompare %}
+
+## How to override the user-agent in your code
+
+### Local testing methods
+
+- **Enable the `chrome://flags/#reduce-user-agent` flag**
+  - This will enable the new behavior and use the reduced format for the
+    user-agent string for all sites.
+- **Configure an emulated device in DevTools with the right user-agent string
+  and client hints**
+  - In DevTools under ⚙️ Settings → Devices → "Add custom device…" you can
+    configure an emulated device with any combination of user-agent string and
+    user-agent client hints values you need. Use the 📱 "Toggle device toolbar"
+    button to select an emulated device.
+- **Launch Chrome with the `--user-agent="Custom string here"`** 
+  - Use the command line flag to start Chrome with a custom user-agent string.
```

Comment:
```suggestion
1. Enable the `chrome://flags/#reduce-user-agent` flag.
    - This will enable the new behavior and use the reduced format for the
    user-agent string for all sites.
2. Configure an emulated device in DevTools with the right user-agent string
  and client hints.
    - In DevTools under ⚙️ **Settings → Devices → Add custom device…** you can
    configure an emulated device with any combination of user-agent string and
    user-agent client hints values you need. Use the 📱 **Toggle device toolbar**
    button to select an emulated device.
3. Launch Chrome with the `--user-agent="Custom string here"`.
    - Use the command line flag to start Chrome with a custom user-agent string.
```

Reply:
Just styleguide stuff, code shouldn't be bold formatted, UI elements should. (Check if it looks alright, I'm editing from GH UI.)

Reply:
Is there a reason you prefer a numbered list here? I thought it made it look more like a sequence as opposed to three different options you could use.

Reply:
The reason is me not reading closely enough, your reasoning is correct. Sorry about that, revert the list, keep the formatting :)

Reply:
Ack, updating.